apps_download/tests/modules/business_requirement/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
```

Comment:
@bizzappdev I think something went wrong with your PR (some bad module copy/paste IMO) :smile:

Reply:
BTW: this is still not fixed @bizzappdev

Reply:
@elicoidal  missed this comment. Please find my explanation.
To run test cases and to create zip file we need real modules.
which can be read from the file system and make a zip file (combine module in case of dependency).

for that test cases, I have intentionally added the modules in that folder

Reply:
:o Great!
Excellent choice :wink: